### PR TITLE
feat: add player translations

### LINKED ISSUE #101: Add player translations

### PR DESCRIPTION
**Pull request by spaenleh**
closes #101

### CODE REVIEW

File: src/langs/en/common.json [hunk 1]
```diff
@@ -8,5 +8,10 @@
   "CLOSE_BUTTON": "Close",
   "AVATAR_DEFAULT_ALT": "avatar",
   "MEMBER_DEFAULT_NAME": "Unknown",
-  "UNKNOWN_DATE": "Unknown"
+  "UNKNOWN_DATE": "Unknown",
+  "USER_OWN_ITEMS": "My Graasp",
+  "USER_SHARED_WITH_ITEMS": "Shared with me",
+  "USER_NOT_SIGNED_IN": "You are not signed in.",
+  "USER_CAN_NOT_ACCESS_ITEM": "You can not access this item.",
```
Comment (pyphilia) [inline src/langs/en/common.json:15]:
```suggestion
"USER_CAN_NOT_ACCESS_ITEM": "You cannot access this item.",
```

File: src/langs/fr/player.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "GRAASP_PLAYER": "Graasp Player",
+  "SHOW_CHAT_TOOLTIP": "Montrer le chat",
+  "HIDE_CHAT_TOOLTIP": "Cacher le chat",
+  "SHOW_PINNED_ITEMS_TOOLTIP": "Montrer les éléments épinglés",
+  "HIDE_PINNED_ITEMS_TOOLTIP": "Cacher les éléments épinglés",
+  "PINNED_ITEMS": "Éléments épinglés",
+  "LOAD_MORE": "Charger plus"
```
Comment (pyphilia) [inline src/langs/fr/player.json:8]:
```suggestion
"LOAD_MORE": "Montrer plus"
```

**Review by pyphilia — APPROVED** [review]
Nice! Thanks for the PR!

**Review by dialexo — APPROVED** [review]
LGTM 🥐 nice! Thank you for setting up i18n on Player 💪

**Pull request merged**